package.json

```diff
@@ -17,7 +17,7 @@
     "lint:md": "markdownlint README.md packages/*/README.md packages/*/demo/**/*.mdx",
     "new": "utils/scripts/new.mjs",
     "prepare": "npm run build",
-    "start": "storybook dev --no-version-updates -p 6006",
+    "start": "npm run build && storybook dev --no-version-updates -p 6006",
```

Comment:
Let's not add this overhead to the `build:demo` and `start` commands.

Reply:
Absolutely. This is only for this spike review in case you want check this locally.